src/main/java/frc/robot/Robot.java

```diff
@@ -22,7 +32,7 @@ public void disabledPeriodic() {
 
     @Override
     public void teleopInit() {
-
+        //new ShooterTeleop(shooterSystem,xboxController).schedule();
```

Comment:
The command should be deleted, so delete this line as well. Generally though, you should avoid leaving code in comments.

Reply:
done